javers-core/src/main/java/org/javers/guava/GuavaCollectionsComparator.java

```diff
@@ -0,0 +1,42 @@
+package org.javers.guava;
+
+import org.javers.core.diff.appenders.CorePropertyChangeAppender;
+import org.javers.core.metamodel.property.Property;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Type;
+
+import static org.javers.core.metamodel.type.JaversType.DEFAULT_TYPE_PARAMETER;
+
+/**
+ * @author akrystian
+ */
+public class GuavaCollectionsComparator{
```

Comment:
albo czegoś tu nie rozumiem albo ta klasa to kopia `CorePropertyChangeAppender`. Zmieniłeś tylko nazwę. O co tu chodzi? Klas się nie kopiuje, to są absolutne podstawy OOP.